packages/server/src/profile/profile.controller.ts

```diff
@@ -99,35 +105,57 @@ export class ProfileController {
   @Post('/upload_picture')
   @UseInterceptors(
     FileInterceptor('file', {
-      dest: 'uploads/',
+      storage: memoryStorage(),
     }),
   )
   async uploadImage(
-    @UploadedFile() file,
+    @UploadedFile() file: Express.Multer.File,
     @User() user: UserModel,
   ): Promise<void> {
     if (user.photoURL) {
-      fs.unlinkSync('uploads/' + user.photoURL);
+      fs.unlink(process.env.UPLOAD_LOCATION + '/' + user.photoURL, (err) => {
+        console.error(
+          'Error deleting previous picture at: ',
+          user.photoURL,
+          err,
+          'the previous image was at an invalid location?',
+        );
+      });
     }
 
-    user.photoURL = file.filename;
-    await user.save();
+    const spaceLeft = await checkDiskSpace('/');
```

Comment:
root dir is diff on windows :( 
use this to detect root dir:
https://stackoverflow.com/a/50299531

Reply:
does this matter since we're on the Khoury VMs?

Reply:
no but in case of windows developers

Reply:
kill all windows developers